sublime_linter.py

```diff
@@ -355,58 +368,61 @@ def group_by_filename_and_update(window, bid, view_has_changed, linter, errors):
         grouped[filename]  # For the side-effect of creating a new empty `list`
 
     did_update_main_view = False
+    window_id = main_view.window().id()
     for filename, errors in grouped.items():
+        for error in errors:
+            error['window'] = window_id
```

Comment:
I don't like this one at all. The `window_id` does not belong to an error. Also the same file can open in multiple windows, and we can maybe support that in the future.